rosdep/base.yaml

```diff
@@ -986,6 +990,9 @@ golang-go:
   gentoo: [dev-lang/go]
   ubuntu: [golang-go]
 google-mock:
+  alpine:
+    source:
+      uri: 'https://github.com/at-wat/alpine-ros-sources/raw/master/googletest/googletest.rdmanifest'
```

Comment:
I see https://pkgs.alpinelinux.org/package/edge/main/x86/gtest-dev which looks like it should resolve the dependency.

Reply:
This gtest-dev package doesn't contain gmock, and gmock is not provided in alpine package repository.

Since [gmock requires to be built with gtest source](https://github.com/google/googletest/blob/b4d4438df9479675a632b2f11125e57133822ece/googlemock/CMakeLists.txt#L15-L19), I think it's not easy to coexist source built gmock with binary installed gtest.
(Patching CMakeLists.txt of gmock might enable to build it using binary installed gtest. I would like to try it.)

Reply:
@tfoote I changed it to use alpine package version of 'gtest' and source rosdep version of 'google-mock'.
(It would be better to register google-mock to upstream apk repository in near future.)